src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -274,8 +273,8 @@ public void customize( IBlockRendering rendering, IItemRendering itemRendering )
 				.rendering( new WirelessRendering() )
 				.build();
 		this.charger = registry.block( "charger", BlockCharger::new )
-				.rendering( new BlockRenderingCustomizer()
-				{
+				.features( AEFeature.Charger )
+				.rendering( new BlockRenderingCustomizer(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)